src/store/index.js

```diff
@@ -0,0 +1,10 @@
+import {combineReducers, createStore, applyMiddleware} from "redux";
+import {Theme, searchFilm} from './reducers/reducers';
```

Comment:
Rename reducer to Film
```suggestion
import {Theme, Films} from './reducers/reducers';
```